test/end-to-end.js

```diff
@@ -434,6 +434,57 @@ describe('end-to-end', function() {
     });
   });
 
+  describe('scaffold 3.x loopback project with option 3.x', function(done) {
+    before(resetWorkspace);
+    before(function createWorkspace(done) {
+      var options = { loopbackVersion: '3.x' };
+      givenWorkspaceFromTemplate('empty-server', options, done);
+    });
+
+    it('contains dependencies with 3.x version', function(done) {
+      var dependencies = readPackageJsonSync().dependencies;
+      expect(dependencies.loopback).to.equal('^3.0.0-alpha.1');
```

Comment:
This test is brittle, it will start failing when we change the scaffolded loopback version to `3.0.0`. Please rework it to use [semver](https://www.npmjs.com/package/semver)'s range matcher instead. Check how @0candy detects whether workspace contains 2.x or 3.x codebase and use a similar approach.